js/main.js

```diff
@@ -301,9 +301,12 @@ function setupFirstTimeLaunch() {
  * @returns {Promise}
  */
 function setStartup(lStartup) {
+    log.send(logLevels.INFO, `launch on startup parameter value is ${lStartup}`);
     return new Promise((resolve) => {
-        let launchOnStartup = (lStartup === 'true');
+        let launchOnStartup = (String(lStartup) === 'true');
```

Comment:
Why not use the `toString()` method?

Reply:
`String(value)` works properly in case of `null` / `undefined` cases whereas `toString()` throws error.